snap/snapcraft.yaml

```diff
@@ -27,3 +28,8 @@ parts:
     plugin: cmake
     build-packages: [build-essential, libboost-all-dev]
     stage-packages: [libicu55]
+    prepare: |
+      if git describe --exact-match --tags 2> /dev/null
+      then
+        touch ../src/prerelease.txt
```

Comment:
We're using `echo -n > prerelease.txt` in other scripts, perhaps that would make sense here too.